backend/alembic/versions/967305fbe7aa_backend_app.py

```diff
@@ -0,0 +1,38 @@
+"""backend_app
+
+Revision ID: 967305fbe7aa
+Revises: fdbceb410b8d
+Create Date: 2023-06-19 15:56:43.145325
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '967305fbe7aa'
+down_revision = '1a9d5e4a1719'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('backendapp',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('app_id', sa.String(), nullable=False),
+    sa.Column('owner', sa.Integer(), nullable=False),
```

Comment:
I think this should not be 1:n but n:n as I think it should be possible to have multiple owners.

Reply:
I was planning to have a separate table for that (in a later PR). There would be one owner, who could invite other developers. But that could be something to discuss.

Reply:
Makes sense, I guess the real question would be, if this will bite us later on / we would need to change it.

Worst I can think of, is admins having to move owners around, as people become unresponsive. And maybe people complaining that there can only be one owner, as they are used to having multiple from github/gitlab I guess.

Reply:
I went ahead and added the extra table (but not the invite system) to this PR.